lib/exercism/user.rb

```diff
@@ -15,8 +15,12 @@ class User < ActiveRecord::Base
   has_many :managed_teams, through: :management_contracts, source: :team
   has_many :team_memberships, -> { where confirmed: true }, dependent: :destroy
   has_many :team_membership_invites, -> { where refused: false }, dependent: :destroy
+  has_many :team_membership_requests, -> { where refused: false }, dependent: :destroy
```

Comment:
Mixed feelings about having a `team_membership_requests` and a `team_membership_requests_denied`.